### PR TITLE
Feat/rush init project default value

### PR DESCRIPTION
**Pull request by nanaSun**
### Basic Checks

Have you run `rush change` for this change?

- [x] Yes
- [ ] No

If **No**, please run `rush change` before, this is necessary.

If adding a **new feature**, the PR's description includes:

- [x] Reason for adding this feature
- [x] How to use
- [x] A basic example

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

### Summary
Take git user.name as default authorName.

### Detail

default git user.name

### CODE REVIEW

File: rush-plugins/rush-init-project-plugin/src/logic/GitConfig.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+export const getGitUserName = (cwd: string): string => {
+
+  const options: child_process.SpawnSyncOptions = {
+    cwd,
+    shell: true,
+    stdio: ['pipe', 'pipe', 'pipe'],
+    env: {}
+  };
+  const result: child_process.SpawnSyncReturns<Buffer> =  child_process.spawnSync(
```
Comment (chengcyber) [inline rush-plugins/rush-init-project-plugin/src/logic/GitConfig.ts:11]: 这里需要加个 try-catch，处理下异常情况

File: rush-plugins/rush-init-project-plugin/src/logic/GitConfig.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import * as child_process from 'child_process';
+
+export const getGitUserName = (cwd: string): string => {
+
+  const options: child_process.SpawnSyncOptions = {
+    cwd,
+    shell: true,
+    stdio: ['pipe', 'pipe', 'pipe'],
```
Comment (chengcyber) [inline rush-plugins/rush-init-project-plugin/src/logic/GitConfig.ts:8]: 这样的话，user name 会打印出来？ 好像不需要打印

File: rush-plugins/rush-init-project-plugin/src/logic/GitConfig.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    env: {}
+  };
+  const result: child_process.SpawnSyncReturns<Buffer> =  child_process.spawnSync(
+    'git',
+    ['config', 'user.name'],
+    options
+  );
+
+  return result.stdout.toString()??'';
```
Comment (chengcyber) [inline rush-plugins/rush-init-project-plugin/src/logic/GitConfig.ts:17]: 感觉缺了 format...

File: rush-plugins/rush-init-project-plugin/src/plopfile.ts [hunk 4]
```diff
@@ -83,6 +84,9 @@ export default function (
       type: "input",
       name: "authorName",
       message: "What is your name? (used in package.json)",
+      default(){
```
Comment (chengcyber) [inline rush-plugins/rush-init-project-plugin/src/plopfile.ts:87]:
```suggestion
      default() {
```

**Review by chengcyber — CHANGES_REQUESTED** [review]

File: rush-plugins/rush-init-project-plugin/src/logic/GitConfig.ts (hunk 3 above)
Comment (nanaSun) [reply rush-plugins/rush-init-project-plugin/src/logic/GitConfig.ts]: done

File: rush-plugins/rush-init-project-plugin/src/plopfile.ts (hunk 4 above)
Comment (nanaSun) [reply rush-plugins/rush-init-project-plugin/src/plopfile.ts]: done

File: rush-plugins/rush-init-project-plugin/src/logic/GitConfig.ts (hunk 1 above)
Comment (nanaSun) [reply rush-plugins/rush-init-project-plugin/src/logic/GitConfig.ts]: done

**Review by chengcyber — APPROVED** [review]

**Pull request merged**